projects/mysadna2/src/app/app.component.scss

```diff
@@ -1,16 +1,20 @@
 .app-navigation {
-    float: right;
-    >a {
-        margin: 5px;
-        width: 83px;
-        height: 24px;
-        font-family: OpenSans;
-        font-size: 18px;
-        font-weight: bold;
-        font-style: normal;
-        font-stretch: normal;
-        line-height: normal;
-        letter-spacing: normal;
-        color: #000000;
-    }
-}
\ No newline at end of file
+  float: right;
+  >a {
+    margin: 5px;
+    width: 83;
```

Comment:
I guess you got me wrong.
The string `width: 83` is invalid. You have to set unit (px, em, pt etc).

In my review I meant: if size of block can be changed from inside, then you shouldn't set size of the block.
Size of `a` element depends on text inside. width and height is redundant here.

Reply:
So what can I do to keep the proportions offered in the mockups?